src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/CompilationArgs.kt

```diff
@@ -172,28 +169,18 @@ class CompilationArgs(
     return this
   }
 
-  fun list(): List<String> = args.toList()
-
-  fun base64Encode(
+  fun flagRepeated(
```

Comment:
The undocumented format changed? Oh, surprise....

This will need to be pulled into the versioned repos. Lmk if you'd prefer to pass this off to me, as I haven't finished writing up how to make the versioned toolchains work.

Reply:
Sure you can take over.

Have an ETA on when the versioned toolchains will be ready? People won't be able to update Kotlin and likely Compose until this is merged.

Reply:
@Bencodes any idea if this works for 1.6-M1? https://github.com/JetBrains/kotlin/releases/tag/v1.6.0-M1 If we can stay on top of the feature previews we can hopefully get 1.6 support out in a timely manner.

Reply:
@jeffzoch In theory yes but I haven't tried yet. We are waiting on some important fixes in 1.6.0 so i'll be looking at supporting that soon.